docs/analytics/quickstart.mdx

```diff
@@ -68,3 +68,14 @@ will become resolved as `parquet` tables become production-ready.
 - [ ] External object store integrations (S3/Azure/GCS/HDFS)
 - [ ] External Apache Iceberg and Delta Lake support
 - [ ] Full text search over `parquet` tables with `pg_bm25`
+
+## For Further Assistance
```

Comment:
Should we link to our Slack here too? We want it as big as possible

Reply:
I intentionally left slack out of a quickstart, since quickstarts should be about code